lib/tls/session.toit

```diff
@@ -136,12 +226,11 @@ class Session:
   session_state -> ByteArray:
     return tls_get_session_ tls_
 
-  internals -> List:
-    return tls_get_internals_ tls_
-
   write data from=0 to=data.size:
     ensure_handshaken_
-    if not tls_: throw "TLS_SOCKET_NOT_CONNECTED"
+    if not tls_:
+      if symmetric_session_: return symmetric_session_.write data from to
+      throw "TLS_SOCKET_NOT_CONNECTED"
```

Comment:
Not entirely sure, but I think I'd prefer always checking `symmetric_session_` before checking `tls_`. Would that work?

```suggestion
    if symmetric_session_: return symmetric_session_.write data from to
    if not tls_: throw "TLS_SOCKET_NOT_CONNECTED"
```